package.json

```diff
@@ -43,7 +43,7 @@
         "shortid": "^2.2.14",
         "video-react": "^0.13.2",
         "vott-ct": "^2.1.19",
-        "vott-react": "^0.2.8"
+        "vott-react": "^0.2.9"
```

Comment:
Looks like package-lock is causing a failure... Not sure how you updated this.  Was it manual? or via `npm install`?  You will have to run a real `npm install` for the package lock to get updated correctly.